irc.html

```diff
@@ -13,21 +13,21 @@ <h1>Channels</h1>
   </p>
   <ul>
     <li>
-      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#minetest</a></strong>
-      (<a href="http://irc.minetest.net/minetest">logs</a>): main channel
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest">#Minetest</a></strong>
+        (<a href="http://irc.minetest.net/minetest">Logs</a>):
+        Main channel where all players and contributors can discuss the game or request support.
     </li>
+    
     <li>
-      <strong>
-        <a href="https://webchat.freenode.net/?channels=#minetest-dev">#minetest-dev</a>
-      </strong>
-      (<a href="http://irc.minetest.net/minetest-dev">logs</a>):
-      development channel, only serious stuff; note that webchat is muted due to too much spam/flood.
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest-dev">#Mminetest-Dev</a></strong>
+        (<a href="http://irc.minetest.net/minetest-dev">Logs</a>):
+        "Development channel, strictly engine or minetest_game development talk only; note that webchat is muted due to too much spam/flood."
     </li>
+    
     <li>
-      <strong>#minetest-hub</strong>
-      (<a href="http://irc.minetest.net/minetest-hub">logs</a>):
-      A moderated channel for people who regularly improve Minetest by developing, modding, hosting servers and otherwise.
-      See <a href="http://hub.minetest.net/info">the channel's website</a> for more information.
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest-hub">#Minetest-Hub</a></strong>
```

Comment:
why is the channel name capitalized?
this does not reflect their actual names on IRC

Reply:
Because Minetest is capitalised and on IRC is up to the IRC client and/or user to "decide" how to represent it. Besides, in my opinion it looks better and more serious when capitalised.

Reply:
It's lower case on the official page: http://hub.minetest.net/info
So it needs to be lower case here.

Reply:
in case it matters: the IRC specification does not mention channel names being case-insensitive, so `#minetest` and `#Minetest` might in theory be totally different channels

Reply:
Roger.